fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Election.java

```diff
@@ -133,19 +138,33 @@ public Map<MessageID, PublicKey> getMessageMap() {
     return messageMap;
   }
 
-  public void putVotesBySender(PublicKey senderPk, List<ElectionVote> votes) {
+  public void putOpenBallotVotesBySender(PublicKey senderPk, List<ElectionVote> votes) {
```

Comment:
I prefer plain as the antonym of encrypted, it's standard in cryptorgraphy discussions. Except if you agreed on all subsystsms to use open ofc

Reply:
what do you mean by antonym?

Reply:
Contrary to. like using plain (and not open) as opposed to encrypted. Like encryptedVote - plainVote (instead of openVote)

Reply:
We already chose the OPEN_BALLOT for describing an open_ballot election. I thought that using the same convention would lead to less confusion.